### DISCUSSION TITLE
Ram increasing

### DISCUSSION THREAD

**Discussion by nezzard** [datarhei/restreamer#566] [Q&A]
Good afternoon, please tell me why the RAM is constantly increasing and sooner or later reaches 100 percent?
I noticed when streams go about 8 hours before ffmpeg takes up almost 100 percent of RAM.
Maybe somewhere in the settings you can fix it?
All streams are configured as rtmp copy video and audio

**Comment by nezzard** [comment]
I went into the container and saw that the main memory consumption goes to ./bin/core
And it increases until the content drops.
[https://pasteboard.co/iB1utKEkbxbq.png](https://pasteboard.co/iB1utKEkbxbq.png)

**Comment by svenerbeck** [comment]
Hello @nezzard

We only process tickets with error logs. It is necessary, especially in your case, to see the error. Here you can see how to get the log file: https://www.youtube.com/watch?v=mZMGrweFCsM — Thank you for your understanding.

Cheerio
Sven

**Reply by nezzard** [reply]
> Hello @nezzard
> 
> We only process tickets with error logs. It is necessary, especially in your case, to see the error. Here you can see how to get the log file: https://www.youtube.com/watch?v=mZMGrweFCsM — Thank you for your understanding.
> 
> Cheerio Sven

[report.txt](https://github.com/datarhei/restreamer/files/11498585/report.txt)
